src/libzippp.h

```diff
@@ -99,14 +99,19 @@ struct zip_source;
 namespace libzippp {
     class ZipEntry;
     class ZipProgressListener;
-    
+    /**
+     * User-defined error-handler.
+     * See https://libzip.org/documentation/zip_error_system_type.html
+     */
+    using ErrorHandlerCallback = std::function<void(int /* error_code_zip */,
+                                                    int /* error_code_system */)>;
```

Comment:
I would suggest to define variables name here, it is more readable IMHO.

```suggestion
    using ErrorHandlerCallback = std::function<void(int zip_error_code,
                                                    int system_error_code)>;
```
Maybe also add a small description about those two codes.